.ci/docker/common/install_executorch.sh

```diff
@@ -37,20 +37,24 @@ install_conda_dependencies() {
 
 install_pip_dependencies() {
   pushd executorch/.ci/docker
+  # Install PyTorch CPU build beforehand to avoid installing the much bigger CUDA
+  # binaries later, ExecuTorch only needs CPU
+  pip_install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
   # Install all Python dependencies
   pip_install -r requirements-ci.txt
   popd
 }
 
 setup_executorch() {
   pushd executorch
-  source .ci/scripts/utils.sh
+  # Setup swiftshader and Vulkan SDK which are required to build the Vulkan delegate
+  as_jenkins bash .ci/scripts/setup-vulkan-linux-deps.sh
 
-  install_flatc_from_source
```

Comment:
why remove this?

Reply:
I replace it by the simpler step of calling `.ci/scripts/setup-linux.sh cmake` which will setup everything including installing flatc.  I'm trying to keep reusing the script from ET as much as I could here.  For example, if someone add another dependency in `setup-linux.sh`, the same step will be applied to PT CI right away as part of the commit hash update.